### PR TITLE
Rename project to Fybrik

### LINKED ISSUE #19: Rename project

### PR DESCRIPTION
**Pull request by revit13**
This PR handles the renaming of the project and resources to Fybrik.

Closes #19

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -88,7 +88,7 @@ make helm-chart-push
 ```bash
 kubectl create -f hello-world-read-module.yaml -n m4d-system
 ```
-3. Check if `M4DApplication` successfully deployed:
+3. Check if `FybrikApplication` successfully deployed:
 ```bash
 kubectl get m4dmodule hello-world-read-module -n m4d-system
 kubectl describe m4dmodule hello-world-read-module -n m4d-system
```
Comment (Mohammad-nassar10) [inline README.md]: Should the `m4dmodule` be changed to `fybrikmodule`? Also for `m4d-system` to `fybrik-system`?

File: README.md [hunk 2]
```diff
@@ -111,22 +111,22 @@ kubectl apply -f sample_assets/secretBank.yaml
...
     - Define `appInfo.purpose`, `appInfo.role`, and `spec.data`.
     - Change `data.dataSetID` field to the identifier of the asset in the catalog which is in the format `<namespace>/<name>`.
  
-2.  Deploy `M4DApplication` in `default` namespace:
+2.  Deploy `FybrikApplication` in `default` namespace:
 ```bash
-kubectl apply -f m4dapplication.yaml -n default
+kubectl apply -f fybrikapplication.yaml -n default
 ```
 3.  Check if `M4DModule` successfully deployed:
```
Comment (Mohammad-nassar10) [inline README.md]: Should `M4DModule` be changed to `FybrikModule`?

File: hello-world-read-module.yaml [hunk 3]
```diff
@@ -1,5 +1,5 @@
   
-apiVersion: app.m4d.ibm.com/v1alpha1
+apiVersion: app.fybrik.io/v1alpha1
 kind: M4DModule
```
Comment (Mohammad-nassar10) [inline hello-world-read-module.yaml]: Should the kind be changed to `FybrikModule`?

Comment (revit13) [reply hello-world-read-module.yaml]: Fixed

File: README.md (hunk 2 above)
Comment (revit13) [reply README.md]: Thanks Fixed

File: README.md (hunk 1 above)
Comment (revit13) [reply README.md]: Thanks Fixed

**Review by Mohammad-nassar10 — APPROVED** [review]

**Pull request merged**